src/payment/checkout/payment-form/CardHolderInformation.jsx

```diff
@@ -215,7 +215,7 @@ export class CardHolderInformationComponent extends React.Component {
               type="text"
               disabled={disabled}
               autocomplete="postal-code"
-              maxlength="10"
+              maxlength="9"
```

Comment:
What about the dash?

Reply:
This is matching the server constraints right now. If you put in 10 (even on the old page) you'll get an unhandled server error.